### PR TITLE
Translate i18n texts to Portuguese #105

### PR DESCRIPTION
**Pull request by ClaraSantosmf**
# Pull request template
 
I translated the document to Portuguese at pt.messages.json, as well as exported the messages in this file to be auto-loaded.

## Motivation and context

Now the site has a Portuguese version to display.

- [ ] Dependencies. If any, specify:
- [ ] Open issue. If applicable, link:

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Change requires a documentation update

## What is the destination branch of this PR?

- [X] `Main`
- [ ] Other. Specify:

## How has this been tested?

Running the project locally, and accessing http://localhost:8080/?lang=pt it was possible to apply translated texts.

## Checklist:

- [X] My code follows the **[style guidelines](https://github.com/empathyco/x/blob/main/.github/CONTRIBUTING.md#style-guides)** of this project.
- [X] I have performed a **self-review** on my own code.
- [X] I have **commented** my code, particularly in hard-to-understand areas.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate **no new warnings**.
- [ ] I have added **tests** that prove my fix is effective or that my feature works.
- [ ] New and existing **unit tests pass locally** with my changes.

### CODE REVIEW

**Comment by ClaraSantosmf** [comment]
Hello, can you accept my PR with the hacktoberfest tag? please

File: src/i18n/messages/pt.messages.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
+{
+    "base": {
```
[the remark this answers is not in the document]
Comment (herrardo) [reply src/i18n/messages/pt.messages.json:2]: I think if you do `npm install` before committing, the prettier package is installed and then the files are formatted correctly.

[the remark this answers is not in the document]
Comment (herrardo) [reply src/i18n/messages/pt.messages.json:2]: If you want to check how your PR is working follow [these steps](https://github.com/empathyco/x-archetype#how-to-install) and then visit
http://localhost:8080/?uiLang=pt

You can change UI language by using uiLang param in the URL

**Comment by herrardo** [comment]
> Hello, can you accept my PR with the hacktoberfest tag? please

Do you mean just to place #Hacktoberfest in the comment when we approve the PR?

File: src/i18n/messages/pt.messages.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ClaraSantosmf) [reply src/i18n/messages/pt.messages.json:2]: Thanks, I fixed that already

File: src/i18n/messages/pt.messages.json [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        "clear": "Limpar",
+        "placeholder": "O que você está procurando?"
+      },
+      "popularSearches": {
+        "title": "BUSCAS POPULARES"
+      },
+      "historyQueries": {
+        "clear": "Limpar histórico",
+        "removeLabel": "Limpar {suggestion} histórico de consulta",
+        "title": "HISTÓRICO DE CONSULTAS"
```
[the remark this answers is not in the document]
Comment (ClaraSantosmf) [reply src/i18n/messages/pt.messages.json:13]: Yes, the best is "histórico de buscas". I fixed this!

**Comment by ClaraSantosmf** [comment]
Morning!! #Hacktoberfest

**Review by herrardo — CHANGES_REQUESTED** [review]
Thanks for your fixes! Just merge main into your branch to resolve the conflicts and I think you will be done!

**Review by mavmaf — CHANGES_REQUESTED** [review]
Hi Clara, thanks for your contribution!
Might you have merged your branch with a non-updated version of the main branch? Some files were accidentally committed. Please take a look and ensure only those files you've been working on are committed. 
Cheers!

**Comment by herrardo** [comment]
@ClaraSantosmf could ypu have a look to @mavmaf comment?

For some reason you didn't update your branch properly. Thanks!

**Pull request closed without merging**